code/__defines/movement.dm

```diff
@@ -30,4 +30,12 @@
 #define S_SOUTHEAST	"6"
 #define S_SOUTHWEST	"10"
 #define S_UP	"16"
-#define S_DOWN "32"
\ No newline at end of file
+#define S_DOWN "32"
+
+#define run_delay 2
```

Comment:
defines should use allcaps to distinguish them from normal variables

Reply:
done